src/include/ibvwrap.h

```diff
@@ -13,9 +13,12 @@
 #define NCCL_IBVWRAP_H_
 
 #include "core.h"
+#include "profiler.h"
 #include <sys/types.h>
 #include <unistd.h>
 
+extern struct ncclProfRecord* g_currentProfilerRecord;
+
 // Dynamically handle dependencies on IB verbs
```

Comment:
Can we reorder the "Replace ntrace hooks with new profiler" commit be the last one, and add commit description about the pre-required changes that were introduced in previous commits? It would be good to also have a summary of each commit's purpose in the PR top description.

Reply:
Yes - I have attempted to re-export with a bit more descriptive metatext, or will open a new PR as need be.